cmd/api.go

```diff
@@ -32,6 +35,8 @@ var apiServer = &cobra.Command{
 			log.Error(err)
 			os.Exit(1)
 		}
+		ctx := context.Background()
```

Comment:
why not using `func WithCancel` to be able to cancel all the context in one place?

Reply:
You are right I am going to open an issue to move all the serverutil.RunALL to use context and wait groups instead of the current solution